### PR TITLE
Move some type definitions to Flow

### PR DESCRIPTION
**Pull request by ppold**
### Summary of Changes

* Move Breakpoint, BreakpointResult, Location, Source, Frame, Scope from tcomb to Flow.

### Test Plan

Make the types pass with Flow

### CODE REVIEW

File: packages/devtools-client-adapters/types.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+};
+
+/**
+ * BreakpointResult
+ *
+ * @memberof actions/types
+ * @static
+ */
+export type BreakpointResult = {
+    ids: string,
```
Comment (jasonLaster) [inline packages/devtools-client-adapters/types.js]: NOTE: we tried to make **flow** fail by adding this error, but when we run flow we got `no errors`

File: src/types.js [hunk 2]
```diff
@@ -1,6 +1,96 @@
 // @flow
 
 /**
+ * Source File Location
```
Comment (jasonLaster) [inline src/types.js]: we tried doing something like this:

```js
import type { Breakpoint, Source } from "devtools-client-adapters/types";
export type { Breakpoint, Source };
```

This worked to the extent that we didn't have to copy the types and the debugger would mention when a type was missing, but it didn't help us w/r/t to validating the types.

**Comment by jasonLaster** [comment]
CC @jlongster @montogeek

File: packages/devtools-client-adapters/types.js (hunk 1 above)
Comment (jlongster) [inline packages/devtools-client-adapters/types.js]: That's a blocker for this PR. :) We can't merge this until we know it's actually working. If it's not applying the types we have no idea if this PR actually works.

Comment (jasonLaster) [inline packages/devtools-client-adapters/types.js]: 💯  sorry that wasn't clear.

**Comment by jasonLaster** [comment]
ping @ppold will you be able to look at this?

File: src/components/Frames.js [hunk 3]
```diff
@@ -15,7 +15,6 @@ import type { List } from "immutable";
 import type { Frame } from "../types";
 
 if (typeof window == "object") {
```
Comment (wldcordeiro) [inline src/components/Frames.js]: This whole block should be removed not left empty.

**Comment by jasonLaster** [comment]
Hmm, things have moved a bit:
* client-adapter is now here: https://github.com/devtools-html/devtools-core/tree/master/packages/devtools-client-adapters
* i predict that the client adapter stuff will need to be manually re-applied so nothing is lost

**Comment by jasonLaster** [comment]
closing this as most of the work will be done in devtools-core

**Pull request closed without merging**